UE4SS/src/GUI/LiveView.cpp

```diff
@@ -279,6 +280,137 @@ namespace RC::GUI
     };
     FLiveViewDeleteListener FLiveViewDeleteListener::LiveViewDeleteListener{};
 
+    static auto add_bool_filter_to_json(JSON::Array& json_filters, const StringType& filter_name, bool is_enabled) -> void {
+        auto json_object = std::make_unique<JSON::Object>();
+        json_object->new_string(STR("FilterName"), filter_name);
+        auto& filter_data = json_object->new_object(STR("FilterData"));
+        filter_data.new_bool(STR("Enabled"), is_enabled);
+        json_filters.add_object(std::move(json_object));
```

Comment:
```suggestion
        auto& json_object = json_filters.new_object();
        json_object.new_string(STR("FilterName"), filter_name);
        auto& filter_data = json_object.new_object(STR("FilterData"));
        filter_data.new_bool(STR("Enabled"), is_enabled);
```
While this code should function, there is a helper function that avoids explicit memory allocation, dealing with a pointer, and std::move.
Note that I'm not necessarily asking you to change this, because I realize you probably copied it from the watches file code, but I do want you to be aware of this API still.
Feel free to resolve this with the button without applying any changes.